Framework/Nexus/inc/MantidNexus/MuonNexusReader.h

```diff
@@ -31,11 +31,12 @@ class DLLExport MuonNexusReader {
   @date 14/08/2008
   */
 private:
-  std::string nexus_instrument_name; ///< name read from nexus file
-  std::string nexus_samplename;      ///< sample name read from Nexus
-  int nexusLogCount;                 ///< number of NXlog sections read from file
-  std::vector<bool> logType;         ///< true if i'th log is numeric
-  std::vector<std::string> logNames; ///< stores name read from file
+  std::string nexus_instrument_name;            ///< name read from nexus file
```

Comment:
Just a quick tidy up: There is a mix of naming standards here. Can we name the members in a more standard way as you have done for the new `m_periodInformation` variable?